client/my-sites/site-settings/press-this-link.jsx

```diff
@@ -84,8 +85,9 @@ class PressThisLink extends React.Component {
 	}
 
 	render() {
+		const omitProps = [ 'site' ];
 		return (
-			<a {...this.props} href={ this.buildPressThisLink() }>
+			<a { ...omit( this.props, omitProps ) } href={ this.buildPressThisLink() }>
```

Comment:
Do you think there's particular value in assigning the omitted props to a variable vs. simply inlining it?

```js
{ ...omit( this.props, 'site' ) }
```